tests/PathTest.php

```diff
@@ -22,4 +22,27 @@ public function testConcatenateOnWindows()
 
         $this->assertEquals('c:\\my-project/src/kernel.php', $path->concatenate(['c:\\my-project', 'src/', 'kernel.php']));
     }
+
+    /**
+     * @dataProvider providePathsForConcatenation
+     */
+    public function testConcatenate($part1, $part2, $expectedPath)
+    {
+        $path = new Path('');
+
+        $actualPath = $path->concatenate([$part1, $part2]);
+
+        $this->assertEquals($expectedPath, $actualPath);
```

Comment:
`assertSame()`?

Reply:
I can change it, but I chose equals to keep it consistent with the existing test.